packages/client-python/src/rocketride/cli/commands/runtime.py

```
@@ -0,0 +1,866 @@
+    from pathlib import Path
+
+    return Path.home() / 'Library' / 'LaunchAgents' / 'com.rocketride.engine.plist'
+
+
+def _autostart_systemd_path():
+    from pathlib import Path
+
+    return Path.home() / '.config' / 'systemd' / 'user' / 'rocketride-engine.service'
+
+
+def _find_rocketride_executable():
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_find_rocketride_executable`

(ANN202)